applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/prebuckling_strategy.hpp

```diff
@@ -127,13 +125,15 @@ class PrebucklingStrategy
 
         mMaxIteration = MaxIteration;
```

Comment:
This should also be in the parameters

Reply:
I didn't add it before, because I passed only the group "buckling_settings" and mMaxIterations is not part of the buckling settings. But will change it then and pass the entire parameter object?

Reply:
Ok I understand
Then leave it as is, this will need some refactor in any case after #3185